### DISCUSSION TITLE
兄弟们，求这张图的出处

### DISCUSSION THREAD

**Discussion by komhshan** [share121/inter-knot#5702] [公告]
![Image_11576090469124](https://github.com/user-attachments/assets/e1af993a-74b3-416a-9484-0cedcdd0e0a6)
求求兄弟们了

**Comment by coolwide** [comment]
试过了，没找到，等英雄

**Comment by tekon2003** [comment]
Paradise Pussy Proven Pussy - Collapse3rd 频道 --- 낙원 보지는 증명된 보지 - 붕괴3rd 채널
https://arca.live/b/hk3rd/61063566<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >
搜到了，这是一组图，自己找<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by komhshan** [reply]
谢谢你，太感谢你了

**Reply by coolwide** [reply]
加里奥6级了！我们有救了！

**Reply by KongBai000-0** [reply]
好人！
